### ISSUE TITLE
Set up CI to run flow, linter, tests

### ISSUE THREAD

**Issue by taneliang** [MLH-Fellowship/scheduling-profiler-prototype#23]
## Acceptance Criteria

- [x] CI runs linter with no errors (https://github.com/MLH-Fellowship/scheduling-profiler-prototype/pull/24)
- [x] CI runs tests with no errors (https://github.com/MLH-Fellowship/scheduling-profiler-prototype/pull/24)
- [ ] CI runs Flow with no errors (blocked by #38)

**Comment by taneliang** [comment]
Partially done, but CI no longer needed in this project since this repo is being imported into React and we'll use their CI setup #134
